### DISCUSSION TITLE
consumer for all queues

### DISCUSSION THREAD

**Discussion by marcoScrJ** [weyoss/redis-smq#123] [Q&A] [answered]
there isn't a way to create a consumer that consumes messages from all existing queues, is possible to do something like this? In this way every new queue i add to redis will be automatically consumed by this consumer

**Comment by weyoss** [comment]
@marcoScrJ Thank you for opening this issue.

No, it is not the way the MQ queue works and technically is not possible.

**Comment by weyoss** [comment, marked as answer]
A consumer may consume message from many queues as you want but you need manually to construct it to do so. It can not automatically catch messages from every new queue.
